pyensembl/ensembl_release_source.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2015. Mount Sinai School of Medicine
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from os.path import split
+
+from .genome_source import GenomeSource
+from .release_info import MAX_ENSEMBL_RELEASE
+from .url_templates import ENSEMBL_FTP_SERVER, make_gtf_url, make_fasta_url
+
+
+class EnsemblReleaseSource(GenomeSource):
+    """
+    Represents the source (URLs) of an Ensembl release. The superclass
```

Comment:
Yeah, `GenomeSource`. I was trying to clarify how it relates to the superclass, but am fine with just removing that line.